scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -486,14 +684,16 @@ For contrast, let's see how integration performs when using the [`harmony` metho
 Algorithmic differences aside, there are a couple practical differences between `fastMNN` and `harmony` to be aware of:
 
 - Unlike `fastMNN`, `harmony` allows you to specify additional covariates (e.g., technology, patient, diagnosis, etc.)
-  - This requires slightly more setup than what we will show below.
+  - This requires additional setup beyond than we will show below.
 - Unlike `fastMNN`, `harmony` does not calculate corrected expression values nor does it return an SCE object.
 Instead, it starts with an uncorrected PC matrix and directly returns a matrix of corrected PCs.
 
+For input, `harmony` can either take a matrix of normalized expression values, from which it will calculate a PCA matrix which it will then batch-correct, or it can take a PCA matrix directly to perform batch-correction on.
```

Comment:
```suggestion
For input, `harmony` can either take a matrix of normalized expression values, from which it will calculate a PCA matrix to batch-correct, or it can take a PCA matrix directly to perform batch-correction on.
```